### ISSUE TITLE
About Pixel-wise

### ISSUE THREAD

**Issue by jianjiandandande** [fzh0917/STMTrack#3]
May I ask if you have compared the performance difference between Pixel-wise Correlation and Depth-wise Correlation in the process of the experiment？

And their time-consuming?

**Comment by fzh0917** [comment]
Yes. We did this experiment. You can read the section 1.1 of [our supplementary material](https://openaccess.thecvf.com/content/CVPR2021/supplemental/Fu_STMTrack_Template-Free_Visual_CVPR_2021_supplemental.pdf) for more details.
